### PR TITLE
Documented AWS Lesson 1, ECS, EC2, and Fargate

### PR DESCRIPTION
**Pull request by Cwagne17**
PR partially completes [Story 6921](https://app.shortcut.com/clarkcan/story/6921/ops-course-aws-lesson-documentation)

Adds documentation for ECS, EC2, and Fargate.

The documentation adds a description, information about the service, SecurEd's use cases, and important further documentation

### CODE REVIEW

**Review by JosueMolinaMorales — APPROVED** [review]
Looks great

File: Ops/Ops-lessons-2022/AWS.md [hunk 1]
```diff
@@ -19,9 +19,73 @@
...
+
+- Serverless
+- Autoscales compute resources
+- No infrastructure management
+- Image -> Define Compute resources -> Run, Manage applications -> Pay
+
+SecurEd Use Cases:
+
+- Scheduled Tasks in ECS Clusters are deployed on Fargate
+- Looking to run low-demand services on Fargate rather then EC2
```
Comment (EVogel1999) [inline Ops/Ops-lessons-2022/AWS.md:142]: I would add that also scalable high-demand services (like bundling) too.

**Review by EVogel1999 — COMMENTED** [review]
Just one small addition to Fargate.  Otherwise looks good!

**Review by pzalep1 — APPROVED** [review]

File: Ops/Ops-lessons-2022/AWS.md (hunk 1 above)
Comment (Cwagne17) [inline Ops/Ops-lessons-2022/AWS.md:142]:
```suggestion
- Looking to run low-demand services on Fargate rather then EC2
- scalable high-demand services
```

**Review by Cwagne17 — COMMENTED** [review]
add

**Review by EVogel1999 — APPROVED** [review]

**Review by StarChaser44 — APPROVED** [review]

**Review by jouypo23 — APPROVED** [review]

**Pull request merged**